### PR TITLE
Issue #1060 New Page for Semester Recap/Year Recap Added

### PR DESCRIPTION
**Pull request by Swatilekha-Roy**
Issue #1060 has been addressed - I have added the 'New Page for Semester Recap/Year Recap" as sem_temp.html. There is still some issues with the .untitled-shutters overflowing below the recap space on my end, but kindly let me know what I can improve or work on further, if at all! Thank you very much. @anushbhatia @smaranjitghose
Screenshot (395)
Screenshot (394)
.

### CODE REVIEW

**Comment by smaranjitghose** [comment]
@Swatilekha-Roy For a first PR to resolve the issue, I must admit this is awesome!



Although, there are certain improvements w.r.t. the code but I would like to integrate this contribution to our codebase and have those for the follow-up PRs

File: sem_temp.html [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
+          <div class="untitled__slideContent">
+            <span>UX Team</span>
+            <span>Happy Hours</span>
+          </div>
+        </div>
+
+        <div class="untitled__slide">
+          <div class="untitled__slideBg"></div>
```
Comment (smaranjitghose) [inline sem_temp.html:137]: We can have this loaded from a sem_temp.js file where the title and the images are defined inside a JSON variable and used in a loop. (Like we have it for the project or experience pages)

**Review by smaranjitghose — APPROVED** [review]

**Comment by smaranjitghose** [comment]
scroll_bar

The size of the scroll bar keeps dynamically changing during the transition. One way maybe to define a separate script for this page exclusively for tackling the issue. However, the more convenient way is to avoid having a scrollbar for this page because the next major section is the footer.

**Comment by smaranjitghose** [comment]
@anushbhatia might be able to help out detailed UI related feedback (if any)

File: sem_temp.html [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        <div class="electron"></div>
+        <div class="electron-alpha"></div>
+        <div class="electron-omega"></div>
+      </div>
+    </div>
+
+    <!--Dynamic Nav Bar-->
+
+    <div class="untitled">
+      <div class="untitled__slides">
```
Comment (smaranjitghose) [inline sem_temp.html:111]: Try to use more appropriate class names 😊

**Comment by Swatilekha-Roy** [comment]
Hi @smaranjitghose @anushbhatia thank you so much for merging the pr and for all your amazing tips!! I will definitely take all your valuable tips into account and make a second pr fixing all these changes by next week!! Thanks again!! 😇

**Pull request merged**